openshift-data-foundation-operator/aggregate/overlays/vsphere-node-labeler/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+commonAnnotations:
+  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
Please remove Argo-specific annotations.  These can be added to a kustomization in your own repo if needed.

Reply:
Removed. This was copied from the aws overlay.